flight_path_viewer.py

```diff
@@ -0,0 +1,75 @@
+from datetime import datetime
+from json import loads
+from fastkml import kml, geometry
```

Comment:
You'll want to add `fastkml` to both `server/requirements.txt` and to pypoetry via `poetry add fastkml`

I've done this already to get it to run - if you'd rather I push this myself, let me know